server/src/codeActions.ts

```diff
@@ -486,6 +499,7 @@ let addUndefinedRecordFieldsV11: codeActionExtractor = ({
       diagnostic,
       file,
       range,
+      todoValue: `%todo`
```

Comment:
Since `%todo` was introduced in v11.1.0, I wanted to add a version check in `addUndefinedRecordFieldsV11`, but passing in `project.rescriptVersion` from `triggerIncrementalCompilationOfFile` all the way through to `addUndefinedRecordFieldsV11` seemed like it wasn't worth the tradeoff in additional complexity